flow2ml/Flow.py

```diff
@@ -205,3 +206,10 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def applyModels(self,algorithms):
+
+    self.algorithms = algorithms
+     
+    if algorithms == 'Logistic_Regression':
```

Comment:
It should be self.algorithms

Reply:
> It should be self.algorithms

There is some problem with taking model as input. I was not able to work on that part. Help me in that part plz.

Reply:
@ainy-123 why do you need model as one input.